modules/profile/general.inc.php

```diff
@@ -370,6 +370,11 @@ public function value(ProfilePage $page, $field, $value, &$success)
                 $success = false;
             } else {
                 $p->assign('email_error', '0');
+                if ($page->values['email_directory'] == 'new@example.org') {
```

Comment:
Done in https://github.com/Polytechnique-org/platal/pull/31/commits/ca2759c51b3f37386282464c887b000e8e779857.